source/PlayerInfo.cpp

```diff
@@ -2820,6 +2821,11 @@ void PlayerInfo::Save(const string &path) const
 	DataWriter out(path);
 
 
+	// Current game version:
+	string version = Format::Split(Format::Split(Files::Read(Files::Resources() + "credits.txt"), "\n")[1], " ")[1];
```

Comment:
As indicated in my other comment; this doesn't seem to be the right source to get the version-number from.

Reply:
This comment is handled.